docs/guides/grafana.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Grafana
+lang: en-US
+meta:
+  - name: keywords
+    content: >-
+      pomerium identity-access-proxy data logging graphing grafana
+      authentication authorization
+description: >-
+  This guide covers how to use Pomerium to authenticate and authorize users of Grafana.
+---
+
+# Securing Grafana with Pomerium
+
+[Grafana](https://grafana.com/) is an open-source analytics and monitoring solution. The many user-contributed [Dashboards](https://grafana.com/grafana/dashboards/) make it popular for enthusiasts as well as professionals.
+
+This guide will demonstrate how to secure an instance of Grafana behind Pomerium, and provide users with a seamless login to Grafana using your Identity Provider (**IdP**).
+
+## Before You Begin
+
+This guide begins with the following steps assumed complete:
+
+- A running instance of Pomerium. If you haven't already installed Pomerium, see our articles for installation on [Docker](/docs/install/readme.md), [Kubernetes](/docs/install/helm.md), or as an all-in-one [binary](/docs/install/binary.html).
+- Administrator access to a working Grafana instance, including write access to the config file `grafani.ini`, usually found in `/etc/grafana`.
```

Comment:
check spelling? is it grafini.ini?

Reply:
@cmo-pomerium where are you seeing `grafini.ini`? Or are you saying maybe it _should_ be `grafini.ini`? If so, I can confirm it's definitely `grafani.ini`.

Reply:
OK, I don't know what I was thinking last night. We were both wrong.

```suggestion
- Administrator access to a working Grafana instance, including write access to the config file `grafana.ini`, usually found in `/etc/grafana`.
```